### PR TITLE
release: v12.1.0

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

This PR marks the feature freeze phase for the `v12.1.0` release.

The current `:edge` image will be tested for roughly **7 days** and if no bugs get reported, `v12.1.0` will be released. During this testing period, only PRs that do not change functionality will be merged.

**If you want to support us, give the latest `:edge` image a try and report any issues you encounter back to this PR.**

Documentation updates may be merged. **Feature freeze is in place until 10 May 2023**.

## Type of change

- [x] Release

## Release-Checklist:

- [x] Update CHANGELOG.md
- [x] Update version in VERSION
- [x] Draft release

### CODE REVIEW

**Review by casperklein — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,60 @@
...
+
+- Rspamd:
+  - note about Rspamd's web interface ([#3245](https://github.com/docker-mailserver/docker-mailserver/pull/3245))
+  - add greylisting option & code refactoring ([#3206](https://github.com/docker-mailserver/docker-mailserver/pull/3206))
+  - added `HFILTER_HOSTNAME_UNKNOWN` and make it configurable ([#3248](https://github.com/docker-mailserver/docker-mailserver/pull/3248))
+  - add option to re-enable `reject_unknown_client_hostname` after #3248 ([#3255](https://github.com/docker-mailserver/docker-mailserver/pull/3255))
+  - add DKIM helper script ([#3286](https://github.com/docker-mailserver/docker-mailserver/pull/3286))
+- make `policyd-spf` configurable ([#3246](https://github.com/docker-mailserver/docker-mailserver/pull/3246))
+- add 'log' command to setup for Fail2Ban ([#3299](https://github.com/docker-mailserver/docker-mailserver/pull/3299))
+- disallow alias = account ([#3270](https://github.com/docker-mailserver/docker-mailserver/pull/3270))
```
Comment (polarathene) [inline CHANGELOG.md:21]: I think that's the correct way to say it :sweat_smile: 

```suggestion
- `setup` command now expects accounts and aliases to be mutually exclusive ([#3270](https://github.com/docker-mailserver/docker-mailserver/pull/3270))
```

**Review by polarathene — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (georglauterbach) [reply CHANGELOG.md:21]: 👍🏼

**Review by polarathene — APPROVED** [review]
That was rather quick :rofl:

**Comment by georglauterbach** [comment]
> That was rather quick 🤣

I was actually about to merge this when your review popped in :D 😄 😆 

I'll merge now :)

**Pull request merged**